project_hr/__manifest__.py

```diff
@@ -1,11 +1,11 @@
 # Copyright 2018 Tecnativa - Pedro M. Baeza
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Project HR',
     'summary': "Link HR with project",
     'development_status': 'Production/Stable',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
This isn't wrong, but nowadays the bot will bump the version number automatically, so it is not really needed.